cub/test/catch2_test_block_radix_sort_keys.cu

```diff
@@ -0,0 +1,164 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Important: Previously, the code was licensed under BSD-3. I don't think we should change that when moving it to a new file.

Reply:
Ah, yeah, good catch: https://github.com/NVIDIA/cccl/pull/6047